options.ts

```diff
@@ -0,0 +1,50 @@
+import { Option } from 'commander';
+import fs from 'fs';
+
+const intParser = (val: string) => parseInt(val);
+
+const getListOfWebsites = (sites: string): string[] => {
+    if (sites) {
+        try {
+            const data = fs
+                .readFileSync(sites, 'utf8')
+                .split('\n')
+                .filter((value) => !!value && !value.startsWith('//'));
+            return data;
+        } catch (err) {
+            console.warn(`Could not read file ${sites}, using ${sites} as a URL`);
+        }
+    }
+    return [sites];
+};
+
+export const keyOption = new Option('-k, --key <key>', 'Add an Google API Key (optional, defaults to no key)');
+
+export const localOption = new Option(
+    '-l, --local',
+    'Run the tests on your local machine (optional, defaults to Google)'
+);
+
+export const numOption = new Option('-n, --num <num>', 'Number of runs (optional, defaults to 5)').argParser(intParser);
```

Comment:
Could we simplify this and use the built-in function?

```suggestion
export const numOption = new Option('-n, --num <num>', 'Number of runs (optional, defaults to 5)').argParser(parseInt);
```

Reply:
Yeah, I realized that I might be able to just do that while re-reading these changes. Will do!